Gemfile

```diff
@@ -41,20 +41,21 @@ gem 'exception_handler', '~> 0.8.0.0'
 # Use unsplash for nice pictures
 gem 'unsplash'
 # use activity_notification as notification system
-gem 'activity_notification', '~> 1.7'
+gem 'activity_notification', '~> 2.2', '>= 2.2.1'
```

Comment:
I moved the lib folder because I thought in Rails 5 the lib folder was auto loaded.
After upgrading to 6 I got constantUninitialized errror for [MemebershipArray](https://github.com/kir-dev/pek-next/blob/master/lib/membership_array.rb) 
I moved the folder back to its original place, but I no longer get the error. Weird.

Reply:
Sometimes `spring` can cause some unexpected weird errors, it's usually worth a try to stop it by calling `bin/spring stop`.